metabase/metabase
# Improve the flow for adding a question to a dashboard

Feature branch for epic [Improve the flow for adding a question to a dashboard](#29666)

### Description
To improve UX of adding question to the dashboard, we added next changes:

- if there was a dashboard visited during last 24hrs, we preselect this dashboard in the ItemPicker by default
- if there wasn't any dashboard visited during last 24hrs, we show root collection

### How to verify

- Go to existing or create a new question
- Click Add to dashboard
- make sure the dashboard you visited recently is preselected in the picker

### Demo

https://www.loom.com/share/41ba6fbd4a2e44b099e9a05206db85a3


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/30347)
